setup.py

```diff
@@ -37,10 +37,10 @@
     ],
     extras_require={
         # for running workflows (in the mgmtworker and the cli), as opposed
-        # to eg. just executing operations (in the agent)
+        # to e.g. just executing operations (in the agent)
         'dispatcher': [
             'PyYAML==6.0',
-            'networkx==1.11',
+            'networkx==2.8.8',
```

Comment:
how about `networkx>2,<3` or something, let's avoid additional strict version pins in setup.py

(will also require rebuilding reqs.txt on the cloudify-manager side, of course)